security/monitoring.adoc

```diff
@@ -0,0 +1,197 @@
+[[security-monitoring]]
+= Monitoring cluster events and logs
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[security-monitoring-intro]]
+== Introduction
+
+In addition to security measures mentioned in other sections of this
+guide, the ability to monitor and audit an {product-title} cluster is an
+important part of safeguarding the cluster itself and its users against
+inappropriate usage.
+
+There are two main sources of cluster-level information that
+are useful for this purpose: events and logs.
+
+[[security-monitoring-events]]
+== Cluster events
```

Comment:
Cluster Events

Reply:
All headings on the page should be in title case

Reply:
Fixed.